core/formats/formats.cpp

```diff
@@ -76,6 +76,8 @@ class format_register
 
 namespace iresearch {
 
+bool noop_memory_accounter(int64_t) noexcept { return true; }
```

Comment:
```suggestion
constexpr bool noop_memory_accounter(int64_t) noexcept { return true; }
```

For example for trivial awaitable stdlib developers writiing something like this